mslice/plotting/plot_window/plot_options.py

```diff
@@ -345,6 +352,14 @@ def __init__(self, line_options, color_validator):
         if self.color_validator is not None:
             self.line_color.currentIndexChanged.connect(lambda selected: self.color_valid(selected))
 
+        if line_options['shown'] is None or line_options['legend'] is None:
+            row5 = QtWidgets.QHBoxLayout()
+            layout.addLayout(row5)
```

Comment:
I think these two lines could go after line 319 (after row3), as we always want row5 at least with delete button. Could they? 
If so, lines 341-342 and 355-357 could be deleted after they are added after line 319!

Reply:
Yes, a line could be deleted even if it is not shown or does not have a legend.